tests/foreman/api/test_errata.py

```diff
@@ -286,15 +308,24 @@ def test_positive_install_multiple_in_host(
     applicable_errata_count = host.content_facet_attributes['errata_counts']['total']
     assert applicable_errata_count > 1
     rhel7_contenthost.add_rex_key(satellite=default_sat)
-    for errata in settings.repos.yum_9.errata[:2]:
-        default_sat.JobInvocation().run(
-            data={
+    for errata in settings.repos.yum_9.errata[1:4]:
+        timestamp = (datetime.utcnow()).strftime('%Y-%m-%d %H:%M')
+        JobInvocation.create(
```

Comment:
Same here, via CLI if we have faster API.

Reply:
Stated above.